src/main/java/com/acoustic/salarycalculator/dbconnection/DatabaseConnection.java

```diff
@@ -0,0 +1,28 @@
+package com.acoustic.salarycalculator.dbconnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+
+public class DatabaseConnection {
+
+    private final Properties properties;
+
+    public DatabaseConnection() {
+        this.properties = new Properties();
+    }
+
+    public Connection getConnection() {
+        try {
+            properties.setProperty("user", DatabaseConfig.USER);
+            properties.setProperty("password", DatabaseConfig.PASSWORD);
+
+            return DriverManager.getConnection(DatabaseConfig.DATABASE_LOCAL_URL, properties);
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
detailed exception -> generic exception this is bad aproach, here just return error message that creating connection failed , some nice description for end user

Reply:
fixed